admin/admin_test.go

```diff
@@ -23,7 +23,7 @@ func TestAdminServerBasicFunctionality(t *testing.T) {
 	testlog := testlog.Logger(t, log.LevelInfo)
 
```

Comment:
can u add a test to see that the result is returned correctly? or alternatively can be in supersim_test

Reply:
added to supersim_test